bin/run-dev.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 ./bin/run-common.sh
-./manage.py runsslserver 0.0.0.0:8000
+./manage.py runserver 0.0.0.0:8000
```

Comment:
Ah, you know what, I think I just misremembered that from MDN, we used a custom hostname, and I think Persona via `localhost` works without SSL. Sorry for the noise :)